example.config.yml

```diff
@@ -155,3 +156,6 @@ php_xdebug_default_enable: 0
 solr_version: "4.10.4"
 solr_xms: "64M"
 solr_xmx: "128M"
+
+# Selenium configuration
+server_version: 2.46.0
```

Comment:
Actually, it looks like the variable _is_ `selenium_version` upstream... is that correct?